src/EndpointsControllers/defaultController.js

```diff
@@ -0,0 +1,59 @@
+import { handleQuery } from '../Helpers/index.js';
+import sql from 'mysql-bricks';
+
+export class DefaultController {
+    constructor (tableName) {
+        this._tableName = tableName;
+    }
+
+    getOne () {
+        return async (req, res) => {
+            const id = req.body.id;
+            const query = sql.select()
+                .from(this._tableName)
+                .where({ id })
+                .toParams({ placeholder: '?' });
+
+            const {
+                values,
+                err
+            } = await handleQuery(query);
```

Comment:
Please, rename `handleQuery` function to something more explanatory, like `getValuesFromQuery`

Reply:
I want to create `handleServerErrors` middleware, i think it will be much more easier to use, so handleQuery will be deleted